kratos/input_output/file_logger_output.cpp

```diff
@@ -0,0 +1,84 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Armin Geiser
+//
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "input_output/file_logger_output.h"
+
+
+namespace Kratos
+{
+    namespace {
+    std::ofstream* CreateNewLogFile(const std::string& rFileName)
+    {
+        // The file stream created here on the heap is deleted in the destructor
+        // of the FileLoggerOutput
+        return new std::ofstream(rFileName);
+    }
+    } // namespace
+
+    FileLoggerOutput::FileLoggerOutput(const std::string& rFileName)
+        : LoggerOutput(*CreateNewLogFile(rFileName))
+    { }
+
+    FileLoggerOutput::~FileLoggerOutput() {
+        delete &GetStream();
+    }
+
+    void FileLoggerOutput::WriteMessage(LoggerMessage const& TheMessage)
+    {
+        auto message_severity = TheMessage.GetSeverity();
+        if (TheMessage.WriteInThisRank() && message_severity <= GetSeverity())
```

Comment:
This part shouldn't be the same as the parent? I mean to decide if writing the severity prefix or not?

Reply:
Yes, but in #6124 i have added the coloring to that check. And that should not happen in the file logger.

I can create a protected function
```cpp
virtual void WriteMessagePrefix()
```
 to the base class to reduce the code duplication.